### PR TITLE
Some janitoring

### PR DESCRIPTION
**Pull request by akva2**
- remove unused private member
- use if constexpr

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/WellInterface_impl.hpp [hunk 1]
```diff
@@ -1181,16 +1181,14 @@ namespace Opm
...
         } else {
-            if (Indices::waterEnabled) {
-                pressure = fs.pressure(FluidSystem::waterPhaseIdx);
+            if constexpr (Indices::waterEnabled) {
+                return fs.pressure(FluidSystem::waterPhaseIdx);
             } else {
-                pressure = fs.pressure(FluidSystem::gasPhaseIdx);
+                return fs.pressure(FluidSystem::gasPhaseIdx);
             }
         }
```
Comment (bska) [inline opm/simulators/wells/WellInterface_impl.hpp:1192]: Could this be
```C++
if constexpr (oilEnabled) {
    return ...
}
else if constexpr (waterEnabled) {
    return ...
}
else {
    return ...
}
```
instead?

Comment (akva2) [reply opm/simulators/wells/WellInterface_impl.hpp:1192]: sure.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks.  I'll merge into master.

**Pull request merged**